src/Hl7.Fhir.Conformance/Model/Generated/CapabilityStatement.cs

```diff
@@ -42,4651 +42,4651 @@
 
 namespace Hl7.Fhir.Model
```

Comment:
This must be a mistake? Why is CS being regenerated here?

Reply:
This was whitespace as a bi-product of merging the develop into the branch (I started from an interim branch that Marco was working on R5 so I could cross test things at the time)